addons/web/static/src/legacy/debug_manager.js

```diff
@@ -0,0 +1,525 @@
+/** @odoo-module **/
+
+import { editModelDebug } from "../core/debug/debug_service";
+import { Dialog } from "../core/dialog/dialog";
+import { formatDateTime, parseDateTime } from "../core/l10n/dates";
+import { _lt } from "../core/l10n/translation";
+import { registry } from "../core/registry";
+import { formatMany2one } from "../fields/format";
+import { json_node_to_xml } from "../views/view_utils";
+
+const { hooks, tags } = owl;
+const { useState } = hooks;
+
+// Action items
+
+function actionSeparator() {
+    return {
+        type: "separator",
+        sequence: 100,
+    };
+}
+
+function accessSeparator({ accessRights, action }) {
+    const { canSeeModelAccess, canSeeRecordRules } = accessRights;
+    if (!action.res_model || (!canSeeModelAccess && !canSeeRecordRules)) {
+        return null;
+    }
+    return {
+        type: "separator",
+        sequence: 200,
+    };
+}
+
+function editAction({ action, env }) {
+    if (!action.id) {
+        return null;
+    }
+    const description = env._t("Edit Action");
+    return {
+        type: "item",
+        description,
+        callback: () => {
+            editModelDebug(env, description, action.type, action.id);
```

Comment:
in case the user does not have access to the `ir.actions.act_window` model (e.g. demo user), this does not crash and loops call to `load_views`

Reply:
reported on the wowl pad